spec/models/shopping_cart_item_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe ShoppingCartItem, type: :model do
+  describe 'validations' do
```

Comment:
Where are the tests for the necessary validations? If not required then lets not have empty method